SU-HSDO/suhumsci
# Add the three_column_w_image template & basic styles to humsci_basic

# READY FOR REVIEW

# Summary
This brings the `three-column-w-image` layout over to the `humsci_basic` theme, and includes some basic layout styles that are applied to the page.

I used as much of the Decanter breakpoint system as I could, (everything but the largest breakpoint).

I opted for a custom flexbox layout over the Decanter one just because our designs used different sizes for the sidebar and gutters. Also, having the sidebars scale down the same rate as the main content wasn't working that well.

# Need Review By (Date)
_1/21_

# Urgency
'high'

# Steps to Test
1. Browse to the page for a News item.
2. Verify that the single-column content is centered at large screen widths, and has screen margins at small screen width.
3. Using layout-builder, add some content to the left-sidebar, save the layout, and confirm that the "sidebar + main content" layout displays appropriately across all screen widths.
4. Using layout-builder, add some content to the right-sidebar, save the layout, and confirm that the "sidebar + main content" layout displays appropriately across all screen widths.

Note: content can be added to both sidebars though that use-case isn't optimized. TBD whether this is out of scope for this issue.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
